Userland/Libraries/LibWeb/CSS/ComputedValues.h

```diff
@@ -340,9 +340,8 @@ inline Gfx::ScalingMode to_gfx_scaling_mode(CSS::ImageRendering css_value, Gfx::
             return Gfx::ScalingMode::BoxSampling;
         return Gfx::ScalingMode::BilinearBlend;
     case CSS::ImageRendering::CrispEdges:
-        return Gfx::ScalingMode::NearestNeighbor;
     case CSS::ImageRendering::Pixelated:
-        return Gfx::ScalingMode::SmoothPixels;
+        return Gfx::ScalingMode::NearestNeighbor;
```

Comment:
Re: discussion in #1170 ; maybe it's a bit nicer to switch around `::CrispEdges` and `::Pixelated` here instead of removing the mapping to `::SmoothPixels`. Then, change the mapping of `::SmoothPixels` in the Skia painter to use nearest neighbour instead.

Reply:
I made a new PR with your idea: https://github.com/LadybirdBrowser/ladybird/pull/1208